src/app/component/comments/components/comments-list/comments-list.component.ts

```diff
@@ -16,6 +16,7 @@ export class CommentsListComponent implements OnInit {
   @Input() public isLoggedIn: boolean;
   @Input() public userId: number;
   @Output() public changedList = new EventEmitter();
+  readonly commentType: string = 'comment';
```

Comment:
Comment type could be only string, isn't it?

Reply:
It should be only string.